EIPS/eip-xxxx.md

```diff
@@ -0,0 +1,72 @@
+---
+
+eip: xxxx  
+title: Parameter Recommendations for Controlled Gas Limit Increase Strategy  
+description: Suggests specific parameters for the controlled gas limit increase strategy introduced in EIP-7783, including starting block number, initial gas limit, rate of increase, and gas limit cap.  
+author: Giulio Rebuffo (@Giulio2002)
+discussions-to: TBD
```

Comment:
Please create a discussions topic in Eth Magicians: https://ethereum-magicians.org/c/eips/5